samples/MyGame/Sample/Color.py

```diff
@@ -0,0 +1,9 @@
+# automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
These files previously weren't part of the repo, as they aren't for most languages? why change this now?